src/wasm/html/index.html

```diff
@@ -14,98 +14,131 @@
+                </div>
+                <p>
+                  <a class="btn btn-primary btn-sm" data-bs-toggle="collapse" href="#collapseLogs" role="button" aria-expanded="false" aria-controls="collapseLogs">
+                    <i class="bi bi-terminal-fill"></i> Show/hide logs
+                  </a>
+                </p>
+            </div>
+          </div>
+          <div class="col-lg-6">
+            <h5>Output</h5>
+            <div class="card">
+                <div class="card-header">
+                  <div class="d-flex w-100 justify-content-between">
+                    <div>
+                        <h5 id="title" class="card-title"></h5>
+                        <h6 id="desc" class="card-subtitle mb-2 text-muted"></h6>
                     </div>
-                    <div id="tree"></div> 
+                    <small id="details">Size: <span id="size">0</span>MB Files: <span id="filesNum">0</span></small>
+                  </div>
+                  <div class="progress mb-2">
+                    <div id="progress-bar" class="progress-bar" role="progressbar" style="width: 0%;">0%</div>
+                  </div>
+                  <div class="mb-2">
+                    <p id="status"> </p>
+                  </div>
+                  <div id="extract-group" class="input-group me-lg-2 w-100">
+                  </div>
                 </div>
+                <div id="tree"></div>
             </div>
+          </div>
         </div>
-    </div>
-    <input type="file" id="fileBrowser" multiple accept=".exe,.bin" style="display:none" />
-    <div id="errorModal" class="modal" tabindex="-1">
-        <div class="modal-dialog">
-          <div class="modal-content">
-            <div class="modal-header">
-              <h5 class="modal-title"><i class="bi bi-exclamation-circle-fill"></i> Error</h5>
-              <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+        <div id="footer" class="col-xl-8 mx-auto p-xl-5 p-3 fixed-bottom">
+          <hr>
+        <small>
+          <a class="collapse-name" onClick='unCollapse("about", this)' role="button">About</a>
+          • <a class="collapse-name" onClick='unCollapse("issues", this)' role="button">Known issues</a>
+          • <a class="collapse-name" onClick='unCollapse("techs", this)' role="button">Open source technologies</a>
+          • <a class="collapse-name" href="https://github.com/Mobica/innoextract-wasm" target="_blank">Github repository</a>
+          <div class="float-end">Innoextract-wasm ${GIT_VERSION}</div>
+          <div id="footer-group">
+            <div id="about" class="collapsible collapsed">
+                <p><a href="https://github.com/Mobica/innoextract-wasm">Innoextract-wasm</a> is created as a side project by <a href="https://mobica.com/">Mobica</a> employees. The main goal was to learn a bit of WASM and porting C/C++ code to a web application using emscripten.<br>
```

Comment:
```suggestion
                <p><a href="https://github.com/Mobica/innoextract-wasm">innoextract-wasm</a> is created as a side project by <a href="https://mobica.com/">Mobica</a> employees. The main goal was to learn a bit of WASM and porting C++ code to a web application using Emscripten.<br>
```

Reply:
Innoextract as-is, capitalized Emscripten.